.github/CODEOWNERS

```diff
@@ -11,7 +11,7 @@
 
 * @iTwin/itwinjs-core-admins
 
-.github/workflows                                 @aruniverse @calebmshafer @wgoehrig @ColinKerr @ben-polinsky
+.github/workflows                                 @aruniverse @calebmshafer @wgoehrig @ColinKerr @ben-polinsky @DanRod1999
```

Comment:
idk if we need all these owners

@calebmshafer , @wgoehrig , @ColinKerr want to be removed from this and from azure-pipelines?